modules/osd-applications-config-custom-domains.adoc

```diff
@@ -0,0 +1,102 @@
+// Module included in the following assemblies for OSD and ROSA:
+//
+// * applications/deployments/osd-config-custom-domains-applications.adoc
+
+[id="osd-applications-config-custom-domains.adoc_{context}"]
+= Configuring custom domains for applications
+
+Custom domains are specific wildcard domains that can be used with {product-title} applications. The top-level domains (TLDs) are owned by the customer that is operating the {product-title} cluster. The Custom Domains Operator sets up a new `ingresscontroller` with a custom certificate as a second day operation. The public DNS record for this `ingresscontroller` can then be used by an external DNS to create a wildcard CNAME record for use with a custom domain.
+
+.Prerequisites
+
+* A user account with `dedicated-admin` privileges
+* A unique wildcard domain, such as ``*.apps.<company_name>.io`
+* A wildcard custom certificate, such as `CN=*.apps.<company_name>.io`
+* Access to a cluster with the latest version of the `oc` CLI installed
+
+[IMPORTANT]
+Do not use the reserved names `default` or `apps*`, such as `apps` or `apps2`, in the `metadata/name:` section of the `CustomDomain` CR.
+
+.Procedure
+
+. Create a new TLS secret from a private key and a public certificate, where `fullchain.pem` and `privkey.pem` are your public or private wildcard certificates.
++
+.Example
++
+
+[source,terminal]
```

Comment:
Two thoughts!

1. AsciiDoc titles will apply to code blocks without the `+`. You should be able to eliminate lines 25 and 26 safely. Same idea throughout the PR. 
2. Do you just want the reader to run the command you have here? If so, you could tweak line 22 a bit, end it with a colon, and eliminate lines 24-26.